mscore/preferences.cpp

```diff
@@ -1149,6 +1151,18 @@ void PreferenceDialog::resetShortcutClicked()
       active->setText(1, shortcut->keysToString());
       shortcutsChanged = true;
       }
+    
+void PreferenceDialog::saveShortcutListClicked()
+      {
+      QString saveFileName = QFileDialog::getSaveFileName(new QWidget(),"Save Shortcuts", tr("XML Files(*.xml)"));
```

Comment:
Where to start .... 
* `new QWidget()`why? Where is this memory freed?
* `tr("XML Files(*.xml)")` Why at this position? This is the filename, not the file type. Do read the the documentation again.
* check the other other usage on `getSaveFileName` in the source code.

Check your spacing, we require a space AFTER a comma.